### PR TITLE
Revert to yarn 1

### PR DESCRIPTION
**Pull request by tmeasday**
For consistency with other apps team projects

<details>
  <summary>📦 Published PR as canary version: <code>0.0.4--canary.13.9d0624d.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.4--canary.13.9d0624d.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.4--canary.13.9d0624d.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,16 +129,10 @@
...
   },
-  "pnpm": {
-    "overrides": {
-      "react": "^18.0.0",
-      "react-dom": "^18.0.0",
-      "@storybook/addon-docs": "^7.1.0-alpha.34",
-      "@storybook/addons": "^7.1.0-alpha.34",
-      "@storybook/api": "^7.1.0-alpha.34",
-      "@storybook/components": "^7.1.0-alpha.34",
-      "@storybook/theming": "^7.1.0-alpha.34"
```
Comment (thafryer) [inline package.json:140]: Should we add these as resolutions as well?

**Review by thafryer — APPROVED** [review]
Small Question. Don't forget to update the readme

File: package.json (hunk 1 above)
Comment (tmeasday) [reply package.json:140]: I don't see why that would be necessary. Perhaps someone can enlighten me. It seems to work without them.

Comment (tmeasday) [reply package.json:140]: I'm also unsure why the react resolutions were there.

**Pull request merged**